site/content/news/rdf4j-420.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "RDF4J 4.2.0 released"
+date: 2022-09-16T21:19:19+0200
+layout: "single"
+categories: ["news"]
+---
+RDF4J 4.2.0 is now available. This is a patch release fixing 8 bugs.
```

Comment:
It's a minor release, right, not a patch one? Might be worth highlighting the performance improvements and of course the custom aggregate feature.

Reply:
It's just the autogenerated stuff at the moment. Haven't had time to sit down and write this up yet.